### PR TITLE
Irminger 9 deployment updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Irminger 9 mooring deployment updates; refer to attached QLR
[3202-00902-Quick-Look-Tables-Irminger-9-AR69-01.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/9047150/3202-00902-Quick-Look-Tables-Irminger-9-AR69-01.pdf)

### CODE REVIEW

File: deployment/GI01SUMO_Deploy.csv [hunk 1]
```diff
@@ -389,53 +389,53 @@ AR60-01,,,,GI01SUMO-SBD12-04-PCO2AA000,8,1,2021-08-12T17:04:00,,CGMGI-01SUMO-000
...
+AR69-01,,,,GI01SUMO-SBD12-00-DCLENG000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGCON-BDCL12-50010,59.936183,-39.470155,,0,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-01-OPTAAD000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-OPTAAD-00256,59.936183,-39.470155,,1,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-02-FLORTD000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-FLORTD-01212,59.936183,-39.470155,,1,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-03-HYDGN0000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-HYDGN0-42529,59.936183,-39.470155,,0,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-04-PCO2AA000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-PCO2AA-34222,59.936183,-39.470155,,0,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-05-WAVSSA000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-WAVSSA-05321,59.936183,-39.470155,,0,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-06-METBKA000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-METLGR-00031,59.936183,-39.470155,,0,2694,,,
+AR69-01,,,,GI01SUMO-SBD12-08-FDCHPA000,9,1,2022-06-30T15:54:00,,CGMGI-01SUMO-00009,,CGINS-FDCHPB-12802,59.936183,-39.470155,,-6,2694,,,
```
Comment (reedan88) [inline deployment/GI01SUMO_Deploy.csv:441]: Looks good.

File: deployment/GI02HYPM_Deploy.csv [hunk 2]
```diff
@@ -67,12 +67,12 @@ AR60-01,,,,GI02HYPM-WFP02-04-CTDPFL000,8,1,2021-08-13T20:46:00,,CGMGI-02HYPM-000
...
+AR69-01,,,,GI02HYPM-WFP02-00-WFPENG000,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGCON-GWFPGX-10401,59.969927,-39.489243,,2560,2676,,,
+AR69-01,,,,GI02HYPM-WFP02-01-FLORDL000,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.969927,-39.489243,,2560,2676,,,
+AR69-01,,,,GI02HYPM-WFP02-03-DOSTAL000,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.969927,-39.489243,,2560,2676,,,
+AR69-01,,,,GI02HYPM-WFP02-04-CTDPFL000,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.969927,-39.489243,,2560,2676,,,
+AR69-01,,,,GI02HYPM-WFP02-05-VEL3DL000,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.969927,-39.489243,,2560,2676,,,
+AR69-01,,,,GI02HYPM-MPM01-02-ZPLSGA009,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,,CGINS-ZPLSGA-55109,59.969927,-39.489243,,150,2676,upward-looking,,
+AR69-01,,,,GI02HYPM-MPM01-02-ZPLSGA010,9,1,2022-07-01T12:49:00,,CGMGI-02HYPM-00009,,CGINS-ZPLSGA-55111,59.969927,-39.489243,,150,2676,downward-looking,,
```
Comment (reedan88) [inline deployment/GI02HYPM_Deploy.csv:78]: Looks good.

File: deployment/GI03FLMA_Deploy.csv [hunk 3]
```diff
@@ -207,29 +207,29 @@ AR60-01,,,,GI03FLMA-RIS01-00-SIOENG000,8,1,2021-08-13T13:22:00,,CGMGI-03FLMA-000
...
+AR69-01,,,,GI03FLMA-RIM01-02-CTDMOG048,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-CTDMOG-13439,59.76685,-39.927187,,500,2690,,,
+AR69-01,,,,GI03FLMA-RIM01-02-CTDMOH049,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-CTDMOH-10229,59.76685,-39.927187,,750,2690,,,
+AR69-01,,,,GI03FLMA-RIM01-02-CTDMOH050,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-CTDMOH-12223,59.76685,-39.927187,,1000,2690,,,
+AR69-01,,,,GI03FLMA-RIM01-02-CTDMOH051,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-CTDMOH-12224,59.76685,-39.927187,,1500,2690,,,
+AR69-01,,,,GI03FLMA-RIS01-00-SIOENG000,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGCON-SIOCON-14884,59.76685,-39.927187,,30,2690,,,
+AR69-01,,,,GI03FLMA-RIS01-03-DOSTAD000,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-DOSTAD-00457,59.76685,-39.927187,,30,2690,,,
+AR69-01,,,,GI03FLMA-RIS01-04-PHSENF000,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-PHSENF-P0181,59.76685,-39.927187,,30,2690,,,
+AR69-01,,,,GI03FLMA-RIS01-05-FLORTD000,9,1,2022-07-03T13:45:00,,CGMGI-03FLMA-00009,,CGINS-FLORTD-01319,59.76685,-39.927187,,30,2690,,,
```
Comment (reedan88) [inline deployment/GI03FLMA_Deploy.csv:235]: Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 4]
```diff
@@ -207,29 +207,29 @@ AR60-01,,,,GI03FLMB-RIS01-00-SIOENG000,8,1,2021-08-17T19:16:00,,CGMGI-03FLMB-000
...
+AR69-01,,,,GI03FLMB-RIM01-02-CTDMOG068,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-CTDMOG-13620,59.71513,-39.317078,,500,2828,,,
+AR69-01,,,,GI03FLMB-RIM01-02-CTDMOH069,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-CTDMOH-12392,59.71513,-39.317078,,750,2828,,,
+AR69-01,,,,GI03FLMB-RIM01-02-CTDMOH070,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-CTDMOH-12390,59.71513,-39.317078,,1000,2828,,,
+AR69-01,,,,GI03FLMB-RIM01-02-CTDMOH071,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-CTDMOH-12222,59.71513,-39.317078,,1500,2828,,,
+AR69-01,,,,GI03FLMB-RIS01-00-SIOENG000,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGCON-SIOCON-12365,59.71513,-39.317078,,30,2828,,,
+AR69-01,,,,GI03FLMB-RIS01-03-DOSTAD000,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-DOSTAD-00507,59.71513,-39.317078,,30,2828,,,
+AR69-01,,,,GI03FLMB-RIS01-04-PHSENF000,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-PHSENF-P0182,59.71513,-39.317078,,30,2828,,,
+AR69-01,,,,GI03FLMB-RIS01-05-FLORTD000,9,1,2022-07-02T13:43:00,,CGMGI-03FLMB-00009,,CGINS-FLORTD-01435,59.71513,-39.317078,,30,2828,,,
```
Comment (reedan88) [inline deployment/GI03FLMB_Deploy.csv:235]: Looks good.

**Review by reedan88 — APPROVED** [review]
Irminger 9 Deploy CSVs look good.

File: deployment/GI03FLMB_Deploy.csv (hunk 4 above)
Comment (cdobs) [inline deployment/GI03FLMB_Deploy.csv:235]: looks good

File: deployment/GI03FLMA_Deploy.csv (hunk 3 above)
Comment (cdobs) [inline deployment/GI03FLMA_Deploy.csv:235]: looks good

File: deployment/GI02HYPM_Deploy.csv (hunk 2 above)
Comment (cdobs) [inline deployment/GI02HYPM_Deploy.csv:78]: looks good

File: deployment/GI01SUMO_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:441]: looks good

**Review by cdobs — APPROVED** [review]
looks good

**Comment by rtravis-whoi** [comment]
[Irminger 9 AM Updates [DT-160]](https://trello.com/c/YNyU7qml/160-irminger-9-am-updates-dt-160)

**Pull request merged**